apps/src/templates/studioHomepages/TopCourse.jsx

```diff
@@ -25,14 +26,19 @@ class TopCourse extends Component {
       lessonName,
       linkToOverview,
       linkToLesson,
-      isRtl
+      isRtl,
+      isProfessionalLearningCourse
     } = this.props;
     const localeStyle = isRtl ? styles.ltr : styles.rtl;
 
     return (
       <div style={styles.card}>
         <img
-          src={require('@cdo/static/small_purple_icons_fullwidth.png')}
+          src={
+            isProfessionalLearningCourse
+              ? require('@cdo/static/small_blue_icons_fullwidth.png')
+              : require('@cdo/static/small_purple_icons_fullwidth.png')
```

Comment:
please try importing these at the top of the file using `import`, like this: https://webpack.js.org/guides/asset-management/
```
import Icon from './icon.png';
```
using `require()` digs us further into a hole which we'll have to dig ourselves out of to make certain webpack optimizations in the future.